include/parallel/parallel_implementation.h

```diff
@@ -1374,6 +1375,147 @@ inline void Communicator::send (const unsigned int dest_processor_id,
 }
 
 
+
+template <typename T>
+inline void Communicator::send (const unsigned int dest_processor_id,
+                                const std::vector<std::vector<T>> & buf,
+                                const MessageTag & tag) const
+{
+  this->send(dest_processor_id, buf,
+             StandardType<T>((buf.empty() || buf.front().empty()) ?
+                             libmesh_nullptr : &(buf.front().front())), tag);
+}
+
+
+
+template <typename T>
+inline void Communicator::send (const unsigned int dest_processor_id,
+                                const std::vector<std::vector<T>> & buf,
+                                Request & req,
+                                const MessageTag & tag) const
+{
+  this->send(dest_processor_id, buf,
+             StandardType<T>((buf.empty() || buf.front().empty()) ?
+                             libmesh_nullptr : &(buf.front().front())), req, tag);
+}
+
+
+
```

Comment:
I know this is the "implementation" part of the file, but it would be helpful (at least for me) if you could put a comment at this point in the file saying which of these overloads are blocking and which are non-blocking.

Reply:
Will do.